manifest/kubernetes/rhel/ncp-rhel.yaml

```diff
@@ -639,7 +650,7 @@ data:
 # Yaml template for NCP Deployment
 # Proper kubernetes API and NSX API parameters, and NCP Docker image
 # must be specified.
-# This yaml file is part of NCP 3.1.1 release.
+# This yaml file is part of NCP 3.1.2 release.
```

Comment:
Do we need to set it to 3.1.1.1? I saw you also included some commits for 3.1.2

Reply:
> Do we need to set it to 3.1.1.1? I saw you also included some commits for 3.1.2 manifest and version syncup

May I ask 3.1.1.1 is a branch or tag on 3.1.1 branch. Thanks. we assume operator and ncp have the same branch strategy from 3.1.2 onward.

Reply:
Frankly I do not have any idea about how this got updated to 3.1.2... yes, I'll set it to 3.1.1.1

Reply:
There is a problem in the way this PR was created! I need to force push it to make sure only my commit is included.

Reply:
Now it's better...